ci_environment/docker/recipes/default.rb

```diff
@@ -18,10 +18,15 @@
   action :add
 end
 
-docker_pkg = 'docker-engine'
-# TODO docker_pkg += "-#{node['docker']['version']}" if node['docker']['version']
-
-package %W(linux-image-extra-#{`uname -r`.chomp} lxc #{docker_pkg})
+package %W(
+  linux-image-extra-#{`uname -r`.chomp}
```

Comment:
I'm pretty confident you don't need this. It's also not going to work until you reboot the machine and re-converge, because this will match the running kernel, not the lower installed one.